omniscidb/QueryEngine/JoinHashTable/Runtime/HashJoinRuntimeCpu.cpp

```diff
@@ -0,0 +1,261 @@
+#include "HashJoinRuntimeCpu.h"
+
+#include <tbb/parallel_for.h>
+
+namespace {
+
+template <ColumnType T>
+inline int64_t getElem(const int8_t* chunk_mem_ptr, size_t elem_size, size_t elem_ind) {
```

Comment:
You better get an error on compile time rather than runtime by using
```
template <ColumnType T>
inline int64_t getElem(const int8_t* chunk_mem_ptr, size_t elem_size, size_t elem_ind) = delete;
```

Reply:
thx, done